README.md

```diff
@@ -224,6 +223,40 @@ If you are inside the Fred Hutch network, you can type
 `rake testlocal` and that will include internal URLs
 in the check.
 
+## Building the site on rhino
+
+This will
+
+### Load a current Ruby version:
+
+```
+ml Ruby/2.7.2-GCCcore-10.2.0
+```
+
+### Install gems
+
+```
+bundle install
+```
+
+### Build
+
+```
+bundle exec jekyll build
+```
+
+### Serve
+
+```
+bundle exec jekyll serve -H <node IP address> -P <port>
```

Comment:
You can use the node hostname (not just IP) or `0.0.0.0`.
Also note that this command will do a build as well, so you can skip the previous (`bundle exec jekyll build`) command if you want.